frappe/contacts/doctype/address/address.py

```diff
@@ -58,7 +58,8 @@ def link_address(self):
 	def validate_reference(self):
 		if self.is_your_company_address:
 			if not [row for row in self.links if row.link_doctype == "Company"]:
-				frappe.throw(_("Company is mandatory, as it is your company address"))
+				frappe.throw(_("Company is mandatory, as it is your company address.\
```

Comment:
A better message would be 

title: "Address not Linked"
message: "You must link this address to a company. Please add a row for {0} in the Links table below."

Reply:
cool